crates/home/src/lib.rs

```diff
@@ -17,12 +8,16 @@
 //! rustup use the standard libraries definition - they use the
 //! definition here.
 //!
-//! This crate further provides two functions, `cargo_home` and
+//! This crate provides two additional functions, `cargo_home` and
 //! `rustup_home`, which are the canonical way to determine the
-//! location that Cargo and rustup store their data.
+//! location that Cargo and rustup use to store their data.
+//! The `env` module contains utilities for mocking the process environment
+//! by Cargo and rustup.
 //!
 //! See also this [discussion].
 //!
+//! See also the [`dirs`](https://docs.rs/dirs) crate.
+//!
```

Comment:
The Cargo project usually avoids mentioning a specific crate in its documentation. I would recommend removing these.

Reply:
Done.

Reply:
Thanks. Could you also bump the crate version to 0.5.6? We might not make a release right away though.